### PR TITLE
Add Boursorama

### PR DESCRIPTION
**Pull request by Jipem**
Boursorama is a French bank and 2FA would be useful.

### CODE REVIEW

**Review by mxxcon — CHANGES_REQUESTED** [review]
Logo must be 32x32 and under 2500 bytes in size.

File: _data/banking.yml [hunk 1]
```diff
@@ -163,6 +163,14 @@ websites:
       phone: Yes
       doc: https://www.mybankofinternet.com/tob/live/usp-core/static/help.html#login_security
 
+    - name: Boursorama
+      url: https://www.boursorama.com/
+      twitter: Boursorama
+      facebook: Boursorama-Banque-1741973942734447
+      img: Boursorama.png
```
Comment (mxxcon) [inline _data/banking.yml]: capitalization matters.
```suggestion
      img: boursorama.png
```

**Review by mxxcon — CHANGES_REQUESTED** [review]

**Review by mxxcon — APPROVED** [review]

**Review by Carlgo11 — APPROVED** [review]

**Comment by Carlgo11** [comment]
Are you sure they don't support 2FA? I haven't see a bank in EU that doesn't support some form of 2FA yet.

**Comment by Jipem** [comment]
@Carlgo11 : 
> Are you sure they don't support 2FA? I haven't see a bank in EU that doesn't support some form of 2FA yet.

Yes, only login + password : https://clients.boursorama.com/connexion/

**Pull request merged**